crates/wash-cli/src/dev.rs

```diff
@@ -1514,7 +1544,7 @@ async fn run_dev_loop(
     let wit_implied_deps = discover_dependencies_from_wit(resolve, world_id)
         .context("failed to resolve dependent components")?;
     eprintln!(
-        "Detected component dependencies: {:?}",
+        "🔎 Detected component dependencies: {:?}",
```

Comment:
`emoji::SEARCH`?

Reply:
I actually fixed this but I must not have force pushed.